diff --git a/node_modules/@astrojs/preact/dist/client.js b/node_modules/@astrojs/preact/dist/client.js
index 32b213a..46efcb5 100644
--- a/node_modules/@astrojs/preact/dist/client.js
+++ b/node_modules/@astrojs/preact/dist/client.js
@@ -19,7 +19,7 @@ var client_default = (element) => async (Component, props, { default: children,
       props[propName] = sharedSignalMap.get(signalId);
     }
   }
-  const bootstrap = client !== "only" ? hydrate : render;
+  const bootstrap = render;
   bootstrap(
     h(Component, props, children != null ? h(StaticHtml, { value: children }) : children),
     element
