diff --git a/node_modules/@astrojs/image/dist/lib/get-image.js b/node_modules/@astrojs/image/dist/lib/get-image.js
index 6c59228..b862e14 100644
--- a/node_modules/@astrojs/image/dist/lib/get-image.js
+++ b/node_modules/@astrojs/image/dist/lib/get-image.js
@@ -76,7 +76,7 @@ async function getImage(transform) {
   }
   const resolved = await resolveTransform(transform);
   const attributes = await loader.getImageAttributes(resolved);
-  const isDev = import.meta.env.DEV;
+  const isDev = import.meta.env?.DEV;
   const isLocalImage = !isRemoteImage(resolved.src);
   const _loader = isDev && isLocalImage ? sharp : loader;
   if (!_loader) {
diff --git a/node_modules/@astrojs/image/package.json b/node_modules/@astrojs/image/package.json
index 7f5824c..df66dc4 100644
--- a/node_modules/@astrojs/image/package.json
+++ b/node_modules/@astrojs/image/package.json
@@ -27,7 +27,7 @@
     "./components": "./components/index.js",
     "./package.json": "./package.json",
     "./client": "./client.d.ts",
-    "./dist/endpoints/*": "./dist/endpoints/*"
+    "./dist/*": "./dist/*"
   },
   "files": [
     "components",
