diff --git a/node_modules/astro/dist/vite-plugin-markdown/index.js b/node_modules/astro/dist/vite-plugin-markdown/index.js
index 451e60e..33c4aab 100644
--- a/node_modules/astro/dist/vite-plugin-markdown/index.js
+++ b/node_modules/astro/dist/vite-plugin-markdown/index.js
@@ -102,7 +102,7 @@ function markdown({ config }) {
         const isPage = fileUrl.pathname.startsWith(resolvePages(config).pathname);
         const hasInjectedScript = isPage && config._ctx.scripts.some((s) => s.stage === "page-ssr");
         let { data: frontmatter, content: markdownContent } = safeMatter(source, filename);
-        markdownContent = markdownContent.replace(/<\s*!--([^-->]*)(.*?)-->/gs, (whole) => `{/*${whole.replace(/\*\//g, "*\u200B/")}*/}`);
+        if (renderOpts.mode === 'mdx') markdownContent = markdownContent.replace(/<\s*!--([^-->]*)(.*?)-->/gs, (whole) => `{/*${whole.replace(/\*\//g, "*\u200B/")}*/}`);
         let renderResult = await renderMarkdown(markdownContent, {
           ...renderOpts,
           fileURL: fileUrl
